src/test/java/org/opensearch/ad/model/ADEntityTaskProfileTests.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
```

Comment:
Note: new files can use the shortened license header. See [here](https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#license-headers)

Reply:
Changed all new files here to include shorted license